File: 11_backbone/burningAirlinesFinal/app/assets/javascripts/models/Flight.js
var app = app || {};

app.Flight = Backbone.Model.extend({
  urlRoot: "/flights",
  default: {
    number: "",
    origin: "",
    destination: "",
    data:"",
    plane:""
  },
  initialize: function() {
    console.log("a new flight was create")
  }
})

File: 08 Rails/bacmd/app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id          :integer          not null, primary key
#  number      :text
#  origin      :text
#  destination :text
#  flightDate  :date
#  plane_id    :integer
#

class Flight < ActiveRecord::Base
  belongs_to :plane
  has_many :reservations
end
